src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -161,13 +162,15 @@ class ActivityFeed extends React.Component<Props, State> {
     };
 
     render(): React.Node {
-        const { feedState, handlers, inputState, isLoading, permissions, translations } = this.props;
+        const { handlers, inputState, isLoading, permissions, translations } = this.props;
         const { approverSelectorContacts, mentionSelectorContacts, isInputOpen } = this.state;
         const { currentUser } = inputState;
         const showApprovalCommentForm = !!(currentUser && getProp(handlers, 'comments.create', false));
         const hasCommentPermission = getProp(permissions, 'comments', false);
         const hasTaskPermission = getProp(permissions, 'tasks', false);
 
+        const feedState = [];
```

Comment:
I am guessing this is just there till you migrate over other areas?

Reply:
yes, that will be filled in this sprint

Reply:
This will override any initial state passed into ContentSidebar for the activity feed. It might be good to have an activity feed in sidebar-activity-feed.html which contains a full suite of comments, tasks, etc that encompass edge cases not likely to be in our the provided file id.

Reply:
Unsure if that means passing down an initialFeedState like what's being done right now or passing initial comments, tasks, etc.